[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/adapters-and-accelerators/accelerator-rosettanet/importing-an-xsd-based-pip.md

### To import an XSD-based PIP

> 1.  Download the XSD-based PIP .zip file from RosettaNet.org or from CIDX.org. Extract the files from the .zip file. The files that you need will be in the subfolders of the XML folder.

Reviewer: ```suggestion
1.  Download the XSD-based PIP .zip file from [GS1 RosettaNet](https://www.gs1us.org/resources/rosettanet). Extract the files from the .zip file. The files that you need will be in the subfolders of the XML folder.
```

### To import an XSD-based PIP

> 1.  Download the XSD-based PIP .zip file from [GS1 RosettaNet website](https://www.gs1us.org/resources/rosettanet) or the [CIDX website](https://oagi.org/pages/chem-estandards).

Reviewer: Combining steps to prevent weird numbering in built version:

```suggestion
1. Download the XSD-based PIP .zip file from [GS1 RosettaNet website](https://www.gs1us.org/resources/rosettanet) or the [CIDX website](https://oagi.org/pages/chem-estandards). Extract the files from the .zip file.

   The files that you need are in the subfolders of the XML folder.
```

> 1. Extract the files from the .zip file. The files that you need will be in the subfolders of the XML folder.

Reviewer: Combine as specified in previous comment:

```suggestion
```
